flake.nix

```diff
@@ -6,9 +6,9 @@
     systems.url = "github:nix-systems/default";
 
     # Backend inputs
-    shared-kernel.url = "github:nammayatri/shared-kernel";
+    shared-kernel.url = "github:nammayatri/shared-kernel/24c7c1c7a23088991f5a537597c0826392910fa1";
     shared-kernel.inputs.nixpkgs.follows = "nixpkgs";
-    beckn-gateway.url = "github:nammayatri/beckn-gateway";
+    beckn-gateway.url = "github:nammayatri/beckn-gateway/1c27df1128687bafaafa4737eeb295288dd6ebf7";
```

Comment:
There shouldn't be a change to `flake.nix`. See https://github.com/nammayatri/nammayatri/tree/main/Backend#updating-flake-inputs

Reply:
Fixed